discovery_engine_flutter/.gitignore

```diff
@@ -6,6 +6,7 @@
 .packages
 build/
 
+ios/Classes/XaynDiscoveryEngineBindings.h
```

Comment:
Do we really need this? We don't need to call these functions from swift anymore.

Reply:
I'm not sure, but didn't we need to provide the headers to iOS so that it gets included
into the binary and not stripped out?

Reply:
If we need the header, then just copying it there is probably not enough.

As I can't really test that we probably should put it into a follow up PR.